packages/cli/src/lib/build-strategies/strategies/DockerVMStrategy.ts

```diff
@@ -136,6 +143,19 @@ export class DockerVMBuildStrategy extends BuildStrategy<void> {
         });
       }
 
+      // TODO: come up with more general solution for this
+      // For JS we need to copy the script file; which may be at root lvl
+      if (language === "wasm/javascript") {
+        // validated by buildManifest manifest validation
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        const scriptFile = buildManifest.config!.scriptFile as string;
```

Comment:
This should be able to be done through a `COPY` command in docker. The manifest properties & config variables are available there.

Reply:
I changed this part so that it's more consistent with what we already do with sources copying. We're keeping the original module file path instead of just its containing directory and using that as "sources" for JS wraps.

That said, a sidenote is we can't copy these with a `COPY` command because the VM's Dockerfile isn't what copies the sources; the sources are copied into a docker shared volume directory that lives in the user's OS. When the VM script is then run inside of the VM container, it assumes the source files it needs are already there and accessible through the shared volume.